assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/calico-host-protection/templates/host-protection.yaml

```diff
@@ -47,7 +47,7 @@ spec:
   - action: Allow
     protocol: TCP
     destination:
-      ports: [6443]
+      ports: [6443, 7443]
```

Comment:
Hmm, why do we need this? I guess we only need 6443 open to the world, right?

Reply:
Ah yes, for the outside we only need 6443. I thought it includes internal NICs too.

Reply:
It's for `bond0` the hostendpoint (at least now, it might change if we use the automatichostendpoint), so if it's on loopback it won't affect

Reply:
Cool! One port not exposed! :)